src/doc/en/developer/packaging.rst

```diff
@@ -359,14 +366,27 @@ at build time,  which should to the appropriate system-specific
   packages should prefix all commands in ``spkg-install.in`` that write into
   the installation hierarchy with ``$SAGE_SUDO``.
 
-Install scripts of script packages
-----------------------------------
+If an ``spkg-src`` file is present, it indicates that the tarball is not
+an unmodified third-party tarball (see :ref:`section-spkg-patching`).
+It documents how the tarball was generated (either by modifying an upstream
+tarball or generating it from a repository). As ideally
+our tarballs are not modified, for most packages there is no ``spkg-src`` file.
+
 
-For script packages, it is also possible to use an install script named ``spkg-install``.
+Install and source scripts of ``script`` packages
+-------------------------------------------------
+
+For ``script`` packages, it is also possible to use an install script named ``spkg-install``.
 It needs to be an executable shell script; it is not subject to the templating
 described in the previous section and will be executed directly from
 the build directory.
 
+Most of our ``script`` packages are associated with a source tree included in the
+repository, in a subdirectory of ``$SAGE_ROOT/pkgs/``. In this case, there
+is a symlink ``src`` that points to the source tree and a script ``spkg-src``
+that builds a tarball for the package.
+
+
```

Comment:
Is this right? The paragraph mentions "tarball"?

Reply:
I meant `spkg-src`.

Reply:
Yes. For example, build/pkgs/sagemath_objects/spkg-src builds the tarball (sdist) of the distribution package sagemath-objects, which can then be uploaded to PyPI.

Reply:
I see. Thanks.